Running-Mastodon/Production-guide.md

```diff
@@ -37,7 +37,9 @@ server {
   server_name example.com;
 
   ssl_protocols TLSv1.2;
-  ssl_ciphers HIGH:!MEDIUM:!LOW:!aNULL:!NULL:!SHA;
+  ssl_ciphers 'ECDHE+CHACHA20:ECDHE+AESGCM'; # or 'ECDHE:!DSS:!PSK:!SRP:!3DES:!RC4:!DES:!IDEA:!RC2:!NULL'
```

Comment:
LibreSSL does not prioritize correctly either :(

So` EECDH+CHACHA20:EECDH+AESGCM:EECDH+AESCCM:EECDH+AES`, which equals to `EECDH+AESGCM:EECDH+AES` for most users seems to be the right choice